src/main/kotlin/calculator/Separates.kt

```diff
@@ -0,0 +1,33 @@
+package calculator
+
+data class Separates(val value: List<String>){
+    fun toIntList(): List<Int> {
+        if(hasValidNumber()){
+            throw RuntimeException("숫자가 아닌 값은 입력될 수 없습니다.")
+        }
+
+        return value.map{
+                number -> number.toInt()
+        }
```

Comment:
```suggestion
        return value.map { number -> 
            number.toInt()
        }
```
람다 관련해서 컨벤션을 적용해보시면 어떨까요?

ref. https://kotlinlang.org/docs/coding-conventions.html#lambdas